### PR TITLE
Milestone3

### PR DESCRIPTION
**Pull request by bellom**
kindly review !

### CODE REVIEW

File: src/component/Button.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+function Button(props) {
+  const { buttonName, color, wide } = props;
+  const buttonColor = color === "grey" ? "grey" : "orange"
+  const buttonWide = wide ? "50%" : ""
+  console.log(wide)
```
Comment (SunnyAsar) [inline src/component/Button.js]: Hi, Please remove log message

File: src/component/Button.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+function Button(props) {
+  const { buttonName, color, wide } = props;
+  const buttonColor = color === "grey" ? "grey" : "orange"
```
Comment (SunnyAsar) [inline src/component/Button.js]: we are supposed to use defaultProps for colour. so any button that doesn't get a prop will automatically use orange.

File: src/component/Display.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+function Display(props) {
+  const { result } = props;
+
+  return (
+    <div className="Display setHeight">
+      <p>{result}0</p>
```
Comment (SunnyAsar) [inline src/component/Display.js]: **Friendly reminder**
This is from the previous milestone, you are supposed to be using defaultProps and propTypes here.

**Review by SunnyAsar — CHANGES_REQUESTED** [review]
# Hi @bellom  👋 
let's use default props to set button colour and remove the log message to make this whole.

**please request another review with the update**

File: src/component/Button.js (hunk 1 above)
Comment (bellom) [reply src/component/Button.js]: forgot ! removed now

**Review by SunnyAsar — APPROVED** [review]
## Looking Good!
### Hey @bellom please proceed.

# You May Pass 👍

**Pull request merged**